src/java/fr/paris/lutece/plugins/lutecetools/service/GitHubService.java

```diff
@@ -120,30 +120,28 @@ public void fill( Component component, StringBuilder sbLogs )
             component.set( GIT_PLATFORM, getGitPlatform( ) );
             Map<String, GHBranch> mapBranches = repo.getBranches( );
             List<String> listBranches = new ArrayList<>( );
+            List<String> extraBranches = new ArrayList<>( );
 
             for ( String strBranch : mapBranches.keySet( ) )
             {
-                listBranches.add( strBranch );
+                if ( "master".equals( strBranch ) || "develop".equals( strBranch ) )
```

Comment:
this list of "main" branches could be in properties ("master" is now currently named  "main")